pkg/app/utils.go

```diff
@@ -163,3 +164,8 @@ func DeleteApp(mgmtAppClient projv3.AppInterface, projectID, appName string) err
 
 	return nil
 }
+
+func isKnownSystemNamespace(ns *v1.Namespace) bool {
+	return slice.ContainsString(strings.Split(settings.SystemNamespaces.Get(), ","), ns.Name, nil) ||
+		rbac.IsFleetNamespace(ns)
```

Comment:
Also probably you don't need `rbac.IsFleetNamespace(ns)` too since they are not added to system namespace anymore

Reply:
Removed.